docker/pulsar-all/pom.xml

```diff
@@ -148,7 +145,18 @@
                         <contextDir>${project.basedir}</contextDir>
                         <tags>
                           <tag>latest</tag>
+                          <tag>${project.version}</tag>
```

Comment:
IIRC this tag is intentionally removed in previous change. cc @nodece

Reply:
We only use the `latest` tag, see https://github.com/apache/pulsar/blob/ed9cca6d5dd947c741b6d6fc579b7d526ba0274c/docker/publish.sh#L65

Reply:
We must use the maven plugin to `push` to build the multi-platform image.

>Buildx is enabled when there is a non-empty <platform> element inside the <buildx> configuration. Only the native platform is built and saved in the local image cache by the build goal. All specified platforms are built and pushed into the remote repository by the push goal. This behavior is to prevent non-native images from tainting the local image cache.
https://dmp.fabric8.io/#build-buildx